lametro/events.py

```diff
@@ -135,6 +163,13 @@ def _merge_events(self, events):
             event['event_details'] = event_details
             event['audio'] = event_audio
 
+        try:
+            assert not spanish_events  # These should all be merged with an English event.
+        except AssertionError:
+            import ipdb; ipdb.set_trace()
+            unpaired_events = [event for event in spanish_events.values()]
```

Comment:
i think it would probably be more appropriate to unpack the tuple here, and pass a list of the dictionaries you want to use to create the exception.